sketches/CAN_haz_bus/CAN_haz_bus.ino

```diff
@@ -72,7 +72,7 @@ void logHexStr(INT32U num, char* prefix, int len)
 
 void setup()
 {
-    Serial.begin(500000);
+    Serial.begin(4000000);
```

Comment:
does this ultimately make a difference?  does this simply hurt devices that actually care?  does this even work with the arduino uno (this firmware is for that board).